src/utils/fetch_json.js

```diff
@@ -1,3 +1,10 @@
+class AjaxError extends Error {
```

Comment:
Is `FetchError` or `HTTPError` better? `AJAX` seems like a less-and-less used term these days. (Don't get me wrong, I love XML.)